vignettes/data-combined.Rmd

```diff
@@ -0,0 +1,300 @@
+observed and/or simulated datasets. Further, some of these datasets can be linked 
+together in a group (e.g., the simulated and the correnspoding observed data)
+for analysis and visualization.
+
+The `DataCombined` class in `{ospsuite}` provides a container to store 
+data, to group them, and to transform them further. The class accepts data 
+coming from `SimulationResults` or as a `DataSet` object (see [Observed data](observed-data.html)).
+
+Let's first generate some simulation results and load observed data.
+
+```{r}
+library(ospsuite)
+
+# Simulation results
+simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
+sim <- loadSimulation(simFilePath)
+simResults <- runSimulation(sim)
+outputPath <- "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)"
+
+# observed data
+obsData <- lapply(
+  c("ObsDataAciclovir_1.pkml", "ObsDataAciclovir_2.pkml", "ObsDataAciclovir_3.pkml"),
+  function(x) loadDataSetFromPKML(system.file("extdata", x, package = "ospsuite"))
+)
+# Name the elements of the list according to the names of the loaded `DataSet`.
+names(obsData) <- lapply(obsData, function(x) x$name)
+```
+
+Typically, `DataCombined` is used to create figures using the `plotXXX()` functions. 
+The different plot types are described in [Visualizations with `DataCombined`](data-combined-plotting.html). 
+To visualize the different functionalities of `DataCombined` in this document, 
+we will use the `plotIndividualTimeProfile()` function.
+
+## Creating `DataCombined` object
+
+First, we create a new instance of `DataCombined` class.
+
+```{r}
+myDataCombined <- DataCombined$new()
+```
+
+Then we add simulated results to this object using `$addSimulationResults()`:
+
+```{r}
+myDataCombined$addSimulationResults(
+  simulationResults = simResults,
+  quantitiesOrPaths = outputPath,
+  names = "Aciclovir Plasma",
+  groups = "Aciclovir PVB"
+)
+```
+
+Next we add observed data to this object using `$addDataSets()`:
+
+```{r}
+myDataCombined$addDataSets(
+  obsData$`Vergin 1995.Iv`, 
+  groups = "Aciclovir PVB"
+)
+```
+
+Every data, be it simulated results or from `DataSet`, must have a unique name 
+within `DataCombined`. If not specified by user, the path of simulated results 
+or the `$name` property of the `DataSet` are used as the name. Alternatively, we 
+can define the name when adding the data, as in the above example adding simulated 
+results.
+
+There are a few things to keep in mind here:
+
+- It doesn't matter in which order observed or simulated datasets are added.
+- You can add multiple `DataSet` objects in `$addDataSets()` method call.
+- You can add only a single instance of `SimulationResults` in `$addSimulationResults()` method call.
+- If you add a dataset with the same name as existing (e.g., adding simulation results 
+from multiple simulations and not specifying the name, so the path of the results 
+is used as the name), the new data will replace the existing.
```

Comment:
I hope user gets a warning in such a case?

Reply:
Nope... Will create an issue for that.

Reply:
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/1127